newlib/libc/tinystdio/conv_flt.c

```diff
@@ -245,7 +245,35 @@ conv_flt (FLT_STREAM *stream, FLT_CONTEXT *context, width_t width, void *addr, u
                 return 0;
 	    }
         }
-	break;
+        if (flt != flt)
```

Comment:
very clever, but perhaps using `isnan` would make this clearer :-)

Reply:
I think it's better to avoid using function from another module as it can cause cascading failures, I can add a comment to make it clearer. What do you think the better option, using `isnan()` or add a comment?   i am okay with both.

Reply:
you're right. let's go with this version.